open_sta_test.tcl

```diff
@@ -0,0 +1,16 @@
+read_liberty /Users/ethanmoon/OpenSTA/sky130hd_tt.lib
```

Comment:
It would e good to rename this to test_verilog_attributes or something descriptive